control-plane-operator/controllers/hostedcontrolplane/oauth/deployment.go

```diff
@@ -134,13 +169,24 @@ func oauthContainerMain() *corev1.Container {
 	}
 }
 
-func buildOAuthContainerMain(image string, noProxy []string) func(c *corev1.Container) {
+func buildOAuthContainerMain(image string, auditWebhookRef *corev1.LocalObjectReference, noProxy []string) func(c *corev1.Container) {
 	return func(c *corev1.Container) {
 		c.Image = image
 		c.Args = []string{
 			"osinserver",
 			fmt.Sprintf("--config=%s", path.Join(volumeMounts.Path(c.Name, oauthVolumeConfig().Name), OAuthServerConfigKey)),
+			"--audit-log-format=json",
+			"--audit-log-maxbackup=1",
+			"--audit-log-maxsize=100",
```

Comment:
I think we should use `10` here like is used for other components.

Reply:
done